uxarray/grid/grid.py

```diff
@@ -1506,6 +1506,18 @@ def global_sphere_coverage(self):
         (i.e. contains no holes)"""
         return not self.partial_sphere_coverage
 
+    @property
+    def inverse_face_indices(self):
+        if self._is_subset:
+            return self._ds["inverse_face_indices"]
+
+    @property
+    def _is_subset(self):
```

Comment:
If we want to make this a property, can we make it public? 
```suggestion
    def is_subset(self):
```